### PR TITLE
Add full static site

### PR DESCRIPTION
**Pull request by eliasjpr**
This adds a demo of a full static website

This demomstrate the capabilities of Amber as a static server.

### CODE REVIEW

File: src/amber.cr [hunk 1]
```diff
@@ -31,15 +31,15 @@ module Amber
       @secret = SecureRandom.hex
     end
 
-    def run(port : Int = 4000)
+    def run
+      puts port
```
Comment (bew) [inline src/amber.cr]: why?

File: src/amber.cr [hunk 2]
```diff
@@ -31,15 +31,15 @@ module Amber
...
       time = Time.now
       host = "127.0.0.1"
-      @port = port.to_i
 
-      str_host = "http://#{host}:#{@port}".colorize(:light_cyan).underline
+      str_host = "http://#{host}:#{port}".colorize(:light_cyan).underline
       version = "[Amber #{Amber::VERSION}]".colorize(:light_cyan).to_s
 
-      log.info "#{version} serving application \"#{@name}\" at #{str_host}".to_s
+      log.info "#{version} serving application \"#{self.name}\" at #{str_host}".to_s
```
Comment (bew) [inline src/amber.cr]: why not just `name`?

File: examples/demo.cr [hunk 3]
```diff
@@ -53,11 +50,11 @@ MyAwesomeApp.config do
     # Each route is defined as follow
     # verb, resources : String, controller : Symbol, action : Symbol,
     # pipeline : Symbol
-    get "/index.html", :hello, :world, :static
+    get "/*", :hello, :world, :static
     get "/hello", :hello, :world, :api
     get "/hello/:planet", :hello, :world, :api
   end
+
+  run
```
Comment (bew) [inline examples/demo.cr]: it's weird to call `run` in the `config` block...

Comment (fridgerator) [inline examples/demo.cr]: agreed

**Pull request merged**